pkgs/tools/security/bitwarden_rs/default.nix

```diff
@@ -8,7 +8,7 @@ let
 
 in rustPlatform.buildRustPackage rec {
   pname = "bitwarden_rs";
-  version = "1.19.0";
+  version = "1.20.0";
 
   src = fetchFromGitHub {
     owner = "dani-garcia";
```

Comment:
You forgot to update the src hash.

Reply:
Sorry, I've done that now.

Reply:
ofborg still fails. not sure why.

Reply:
The log mentions cargo256sha being out of date. I've tried to follow the steps suggested and pushed a new set of commits.